src/validation.ts

```diff
@@ -0,0 +1,34 @@
+import {Migration} from "./types"
+
+const indexNotMatch = (migration: Migration, index: number) =>
+  migration.id !== index
+
+/** Assert migration IDs are consecutive integers */
+export function validateMigrationOrdering(migrations: Array<Migration>) {
```

Comment:
I left both of the functions in this file here, but neither of the are used in our modified version.